test/treegen.lua

```diff
@@ -44,22 +44,31 @@ local function find_template(g, script)
     error(("treegen: can't find a template for script %q"):format(script))
 end
 
--- Generate a script that follows a template and write it at given
--- script file path in given directory.
-local function write_script(g, dir, script, replacements)
+-- Write provided script into given directory.
+function treegen.write_script(dir, script, body)
     local script_abspath = fio.pathjoin(dir, script)
     local flags = {'O_CREAT', 'O_WRONLY', 'O_TRUNC'}
     local mode = tonumber('644', 8)
-    local fh = fio.open(script_abspath, flags, mode)
-    local template = find_template(g, script)
-    local replacements = fun.chain({script = script}, replacements):tomap()
 
-    log.info(('Writing a script: %s'):format(script_abspath))
-    fh:write(template:gsub('<(.-)>', replacements))
+    local scriptdir_abspath = fio.dirname(script_abspath)
+    log.info(('Creating a directory: %s'):format(scriptdir_abspath))
+    fio.mktree(scriptdir_abspath)
 
+    log.info(('Writing a script: %s'):format(script_abspath))
+    local fh = fio.open(script_abspath, flags, mode)
+    fh:write(body)
     fh:close()
 end
 
+-- Generate a script that follows a template and write it at given
+-- script file path in given directory.
```

Comment:
Typo: s/in given directory/in the given directory/

Reply:
Fixed (and all similar typos around). Thank you!